tools/pg_tools/pg-query.xml

```diff
@@ -0,0 +1,48 @@
+<tool id="pg_query" name="Postgresql" version="@PG_VERSION@">
+    <description>import sql dump</description>
```

Comment:
description

Reply:
?

Reply:
change description from "import sql dump" to something like "query postgres database"